habitat-lab/habitat/config/benchmark/rearrange/cat_pick.yaml

```diff
@@ -29,9 +29,21 @@ habitat:
     agents:
       main_agent:
         radius: 0.3
-        robot_urdf: ./data/robots/hab_fetch/robots/hab_suction.urdf
-        robot_type: FetchSuctionRobot
-
+        robot_urdf: data/robots/hab_stretch/urdf/hab_stretch.urdf
+        robot_type: "StretchRobot"
+        sim_sensors:
+           head_rgb_sensor:
+             height: 256 # 640
```

Comment:
Thanks for this! Since it is in simulation, is it okay to use the parameters that are different from the real hardware? I think it is okay with since we are running this in simulation.

Reply:
Yes, we will continue using these parameters for the current experiments.

Reply:
@jimmytyyang We are using these parameters only for the time we want to do some quick experimentation. Paper results will be with the original parameters.